src/api/types/DexDtos.ts

```diff
@@ -1281,3 +1294,63 @@ export class SetGlobalLimitOrderConfigDto extends SubmitCallDTO {
     this.uniqueKey = data?.uniqueKey;
   }
 }
+
+export class MakePoolPublicDto extends SubmitCallDTO {
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token0: TokenClassKey;
+
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token1: TokenClassKey;
+
+  @EnumProperty(DexFeePercentageTypes)
+  public fee: DexFeePercentageTypes;
+
+  constructor(token0: TokenClassKey, token1: TokenClassKey, fee: DexFeePercentageTypes) {
+    super();
+    this.token0 = token0;
+    this.token1 = token1;
+    this.fee = fee;
+  }
+}
+
+export class ManageWhitelistDto extends SubmitCallDTO {
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token0: TokenClassKey;
+
+  @IsNotEmpty()
+  @ValidateNested()
+  @Type(() => TokenClassKey)
+  public token1: TokenClassKey;
+
+  @EnumProperty(DexFeePercentageTypes)
+  public fee: DexFeePercentageTypes;
+
+  @IsNotEmpty()
+  @IsString()
+  public targetUser: string;
+
+  @IsNotEmpty()
+  @IsBoolean()
+  public isAdd: boolean; // true to add, false to remove
```

Comment:
what about

```ts
export enum PoolWhitelistOperation {
  ADD = "ADD",
  REMOVE = "REMOVE";
}
```

and then:


```suggestion
  @IsNotEmpty()
  @StringEnumProperty(PoolWhitelistOperation)
  public operation?: PoolWhitelistOperation;
```